src/test/java/io/stargate/sgv2/jsonapi/service/operation/model/impl/DeleteOperationTest.java

```diff
@@ -52,12 +52,14 @@
 public class DeleteOperationTest extends OperationTestBase {
   private static final String KEYSPACE_NAME = RandomStringUtils.randomAlphanumeric(16);
   private static final String COLLECTION_NAME = RandomStringUtils.randomAlphanumeric(16);
-  private static final CommandContext COMMAND_CONTEXT =
-      new CommandContext(KEYSPACE_NAME, COLLECTION_NAME);
-
-  @Inject QueryExecutor queryExecutor;
+  private CommandContext COMMAND_CONTEXT;
   @Inject ObjectMapper objectMapper;
-  @Inject QueriesConfig queriesConfig;
+
+  @PostConstruct
+  public void init() {
+    COMMAND_CONTEXT =
+        new CommandContext(KEYSPACE_NAME, COLLECTION_NAME, "testCommand", jsonBytesMetricsReporter);
+  }
 
```

Comment:
In Java CDI, field injection (inject `jsonBytesMetricsReporter` in `OperationTestBase`) happens after the constructor execution. So I have to instantiate `COMMAND_CONTEXT` after the constructors execute. Same for below.

Reply:
I added a note later on, but maybe makes sense to suggest here: since `CommandContext` can no longer be created eagerly, early on, there's no point in creating `COMMAND_CONTEXT` eagerly.
Instead you can just create helper factory method like `createCommandContext()` that constructs and returns this instance when test needs it.
These are cheap objects to create so there isn't much benefit from constructing them eagerly anyways (or reusing).